### DISCUSSION TITLE
Can I order the level shifter board?

### DISCUSSION THREAD

**Discussion by Koohiisan** [darthcloud/BlueRetro#121] [Q&A] [answered]
I see it is used in a number of system configurations.  Is it something I can buy?  Or is it necessary to build it entirely from scratch?  I didn't see anything on the wiki, so asking here.

Thanks!

**Comment by darthcloud** [comment, marked as answer]
I'm not selling anything unfortunately, you can find the geber here in the zip files https://github.com/darthcloud/BlueRetroHW/releases

You can directly upload the zip to a pcb fab like jlcpcb or pcbway to get them fabricated.

**Reply by Koohiisan** [reply]
Okay, that makes sense.  I've never done this kind of fabrication before...I assume it's just the PCB with me having to solder the bits and pieces onto it, right?

**Reply by darthcloud** [reply]
Yes, but if you didn't do this before it will be hard. Its very small parts to solder.

**Reply by Koohiisan** [reply]
I have soldered surface mount components before, if that's the type of size I'd be dealing with, but I've never soldered this many.  I'd probably get a few boards made and if something goes terribly wrong I can just practice with that one.
